frontend/src/scss/PrimeStyles.scss

```diff
@@ -2,6 +2,10 @@
   background-color: color("base-lightest");
   padding-top: units(2);
   padding-bottom: units(2);
+
+  .grid-container {
+    max-width: 75rem;
```

Comment:
could this technically not resolve to 1200px? depends on font size? I wonder if we'd want it to match the size of `.usa-nav-container.prime-header` even if folks are using a browser with a non-standard font size

Reply:
Good catch 👍 [bd4f1bd](https://github.com/CDCgov/prime-simplereport/pull/4759/commits/bd4f1bdfdec759b4564ff262f727df4615c99f3f) sets this to the same constant 1200px as the header.